src/providers/authenticationReducer.ts

```diff
@@ -12,8 +13,10 @@ export const authenticationReducer: AuthenticationReducer = (
   switch (action.type) {
     case 'set':
       return { token: action.token };
-    case 'clear':
-      return { token: "" };
+    case 'clear': {
+      localStorage.setItem(localStorageKey, '');
```

Comment:
Why not to [remove](https://developer.mozilla.org/en-US/docs/Web/API/Storage/removeItem) the item from the storage entirely?